libs/damap/src/lib/widgets/license-wizard/license-wizard-list.ts

```diff
@@ -275,6 +302,7 @@ export const bsd2c = {
   compatibility: ['bsd-2c', 'bsd-3c', 'apache-2', 'lgpl-2.1', 'lgpl-2.1+', 'lgpl-3', 'mpl-2', 'epl-1', 'cddl-1', 'gpl-2', 'gpl-2+', 'gpl-3', 'agpl-1', 'agpl-3']
 };
 export const apache2 = {
+  id: 'apache2',
   name: 'license.wizard.license.apache2.name',
   priority: 4,
   available: true,
```

Comment:
general question: does it make sense to have the licenses in the backend and in the frontend? Frontend should probably fetch available licenses, so it is easier to manage and updates only have to be performed in one place (single source of truth).

Other than that: depending on the name change of the enum values in the backend PR, this might need an update.

Reply:
yes it would make sense, but the current implementation has a very large feature fully implemented into the frontend, the license wizard, which is utilizing all the additional list properties in this file.
refactoring it into the backend would be a lot more effort tha it is worth.

Reply:
Understandable. Might be worth refactoring in a future issue